src/mudlet.cpp

```diff
@@ -3917,7 +3920,11 @@ QString mudlet::autodetectPreferredLanguage()
         }
     }
 
-    for (auto language : QLocale::system().uiLanguages()) {
+    QStringList systemUiLanguages = QLocale::system().uiLanguages();
+    // The code in the loop may modify the value anyway so explicity detach
+    // from the original:
+    systemUiLanguages.detach();
+    for (auto& language : systemUiLanguages) {
```

Comment:
The original code was producing that warning about an iterator detaching from the container it was iterating over - but this shuts it up (because of what happens in the loop it is not possible to make the iterator be a `const` reference and/or add the `qAsConst(...)` wrapper around the container).